### PR TITLE
Small typos in doc

### PR DESCRIPTION
**Pull request by Riuqlav**
Small typos in the doc, I did not modify any code.

### CODE REVIEW

**Review by dougsillars — CHANGES_REQUESTED** [review]
Thanks for the edits - these are great!

I have one change to request however:

in the introduction.md = all your edits are fine -except "What is Conductor" is grammatically correct, whereas your change of "what is the conductor" is not (Conductor is the product we are talking about)

**Comment by Riuqlav** [comment]
Some more small ones

File: docs/reference-docs/fork-task.md [hunk 1]
```diff
@@ -11,19 +12,19 @@ sidebar_position: 1
 
 A Fork operation in conductor, lets you run a specified list of other tasks or sub workflows in parallel. A fork task is
 followed by a join operation that waits on the forked tasks or sub workflows to finish. The `JOIN`
-task also collects outputs from each of the forked tasks or sub workflows.
+the task also collects outputs from each of the forked tasks or sub workflows.
```
Comment (dougsillars) [inline docs/reference-docs/fork-task.md:15]: This change will now read "The JOIN the task also collects outputs from each of the forked tasks or sub workflows.

The old version was "The `JOIN`task also collects outputs from each of the forked tasks or sub workflows."

I think the old version is correct

File: docs/reference-docs/do-while-task.md [hunk 2]
```diff
@@ -3,82 +3,86 @@ sidebar_position: 1
...
 ```
+
 ## Introduction
-Sequentially execute a list of task as long as a condition is true. 
+
+Sequentially execute a list of task as long as a condition is true.
 The list of tasks is executed first, before the condition is checked (even for the first iteration).
 
-When scheduled, each task of this loop will see its `taskReferenceName` concatenated with __i, with i being the iteration number, starting at 1. Warning: taskReferenceName containing arithmetic operators must not be used.
+When scheduled, each task of this loop will see its `taskReferenceName` concatenated with \_\_i, with i being the iteration number, starting at 1. Warning: taskReferenceName containing arithmetic operators must not be used.
```
Comment (dougsillars) [inline docs/reference-docs/do-while-task.md:16]: We don't need to escape the characters in the docs - it adds confusion to the reader (IMO)

File: docs/reference-docs/do-while-task.md [hunk 3]
```diff
@@ -3,82 +3,86 @@ sidebar_position: 1
...
-### Limitations 
-- Domain or isolation group execution is unsupported; - Nested DO_WHILE is unsupported; 
-- SUB_WORKFLOW is unsupported; 
-- Since loopover tasks will be executed in loop inside scope of parent do while task, crossing branching outside of DO_WHILE task is not respected. 
-
-Branching inside loopover task is supported.
+### Limitations
 
+- Domain or isolation group execution is unsupported; - Nested DO_WHILE is unsupported;
+- SUB_WORKFLOW is unsupported;
```
Comment (dougsillars) [inline docs/reference-docs/do-while-task.md:25]: this is an error.. can you please remove this line (subworkflows are supported)

File: docs/reference-docs/dynamic-fork-task.md [hunk 4]
```diff
@@ -3,15 +3,16 @@ sidebar_position: 1
...
 # Dynamic Fork
+
 ```json
 "type" : "FORK_JOIN_DYNAMIC"
 ```
 
 ## Introduction
 
-A Fork operation in conductor, lets you run a specified list of other tasks or sub workflows in parallel after the fork
+A Fork operation in the conductor, lets you run a specified list of other tasks or sub workflows in parallel after the forked
```
Comment (dougsillars) [inline docs/reference-docs/dynamic-fork-task.md:13]: Conductor is the name of the tool.  
I think perhaps:
"A Fork operation in Conductor lets you run a specified list of other tasks..."
reads better (capitalized Conductor and removed the comma)

File: docs/reference-docs/dynamic-fork-task.md [hunk 5]
```diff
@@ -3,15 +3,16 @@ sidebar_position: 1
...
 task. A fork task is followed by a join operation that waits on the forked tasks or sub workflows to finish. The `JOIN`
-task also collects outputs from each of the forked tasks or sub workflows.
+the task also collects outputs from each of the forked tasks or sub workflows.
```
Comment (dougsillars) [inline docs/reference-docs/dynamic-fork-task.md:15]: "The `JOIN` the task also collects.."

I don't think this is correct.

File: docs/reference-docs/dynamic-fork-task.md [hunk 6]
```diff
@@ -21,16 +22,16 @@ There are three things that are needed to configure a `FORK_JOIN_DYNAMIC` task.
...
 
 ## Use Cases
 
-A `FORK_JOIN_DYNAMIC` is useful, when a set of tasks or sub-workflows needs to be executed and the number of tasks or
+A `FORK_JOIN_DYNAMIC` is useful when a set of tasks or sub-workflows needs to be executed and the number of tasks or
 sub-workflows are determined at run time. E.g. Let's say we have a task that resizes an image, and we need to create a
 workflow that will resize an image into multiple sizes. In this case, a task can be created prior to
 the `FORK_JOIN_DYNAMIC` task that will prepare the input that needs to be passed into the `FORK_JOIN_DYNAMIC` task. The
-single image resize task does one job. The `FORK_JOIN_DYNAMIC` and the following `JOIN` will manage the multiple
+single image resizes task does one job. The `FORK_JOIN_DYNAMIC` and the following `JOIN` will manage the multiple
```
Comment (dougsillars) [inline docs/reference-docs/dynamic-fork-task.md:34]: I think this should be singular.  the task does a resize on just one image.  single image resize.

**Review by dougsillars — CHANGES_REQUESTED** [review]
These are all great - thank you for your efforts in making our docs better.
I left a few more comments/suggestions.

**Pull request closed without merging**